.env.k8s.mauve-dev

```diff
@@ -20,3 +20,4 @@ VIOLET_ENV="development"
 REACT_APP_VIOLET_ENV="development"
 REACT_APP_VIOLET_CLIENT_ID="e9f7df20fe2c0b87b431477a604151dcea3b8ee1dee16125b46ec62c0c173a24"
 REACT_APP_VERSION="v1.0.0"
+REACT_APP_SEGMENT_WRITE_KEY="M2XN7OZIHMQjRih3HzaXqT9ud76hO12K"
```

Comment:
Are the segment write keys supposed to be the same for all environments?

Reply:
Yup, for now we are not paying and only have one source, we will be using the same one for all environments until we finish setting everything up and testing on dev environment, then we have the option to either pay for separate environments or use only on PROD